src/installDeps.ts

```diff
@@ -33,7 +33,7 @@ export async function findManifestAndInstallDeps(
     let installType: DepsInstallType | undefined;
 
     if (manifestFileInput) {
-        manifestDir = path.join(manifestDirInput, path.dirname(manifestFileInput));
+        manifestDir = path.join(manifestDirInput, manifestFileInput);
         manifestFilename = path.basename(manifestFileInput);
```

Comment:
I think output of `path.basename(manifestFileInput)` will always be `manifestFileInput`. As input `manifest_file` will only contain file name.